.travis.yml

```diff
@@ -15,6 +15,7 @@ env:
     - REPO=crops/toaster-krogoth BRANCH=krogoth DOCKERHUB_TAG="latest" POKYBRANCH=krogoth
     - REPO=crops/toaster-krogoth BRANCH=yocto-2.1 DOCKERHUB_TAG="2.1" POKYBRANCH=krogoth
     - REPO=crops/toaster-krogoth BRANCH=yocto-2.1.1 DOCKERHUB_TAG="2.1.1" POKYBRANCH=krogoth
+    - REPO=crops/toaster-krogoth BRANCH=yocto-2.1.2 DOCKERHUB_TAG="2.1.2" POKYBRANCH=krogoth
```

Comment:
I put it in the other review as well, but this should be a separate commit, because it doesn't have anything to do with updating to morty.